From bd0e39e64d0dd7a81f0129f790ee695fb396a0fe Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 25 May 2023 22:29:29 +1100
Subject: [PATCH 06/11] no more Release/ subfolder

---
 faiss/python/setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/faiss/python/setup.py b/faiss/python/setup.py
index 23611cb3..4745b970 100644
--- a/faiss/python/setup.py
+++ b/faiss/python/setup.py
@@ -23,7 +23,7 @@ shutil.copyfile("extra_wrappers.py", "faiss/extra_wrappers.py")
 shutil.copyfile("array_conversions.py", "faiss/array_conversions.py")
 
 ext = ".pyd" if platform.system() == "Windows" else ".so"
-prefix = "Release/" * (platform.system() == "Windows")
+prefix = ""
 
 swigfaiss_generic_lib = f"{prefix}_swigfaiss{ext}"
 swigfaiss_avx2_lib = f"{prefix}_swigfaiss_avx2{ext}"
