From 5e576fb95233970295084e7d05b6818d7667af6a Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 8 Oct 2020 18:06:20 +0200
Subject: [PATCH 7/7] use c++14

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 31b587b..5755fb6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -16,7 +16,7 @@ AC_CONFIG_MACRO_DIR([acinclude])
 # Checks for programs.
 AC_LANG(C++)
 AC_PROG_CXX
-AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory])
+AX_CXX_COMPILE_STDCXX([14], [noext], [mandatory])
 AC_PROG_CPP
 AC_PROG_MAKE_SET
 AC_PROG_MKDIR_P
-- 
2.26.2.windows.1

