From fb6103d0d1f044c8b27450d93f1d1d6f82c2c1b4 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 9 Oct 2020 18:24:57 +0200
Subject: [PATCH 8/8] backport facebookresearch/faiss#1388

---
 gpu/StandardGpuResources.cpp | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/gpu/StandardGpuResources.cpp b/gpu/StandardGpuResources.cpp
index e564f8e..257fc2a 100644
--- a/gpu/StandardGpuResources.cpp
+++ b/gpu/StandardGpuResources.cpp
@@ -249,11 +249,15 @@ StandardGpuResources::initializeForDevice(int device) {
   blasHandles_[device] = blasHandle;
 
   // Enable tensor core support if available
-#if CUDA_VERSION >= 9000
+#if CUDA_VERSION >= 9000 && CUDA_VERSION < 11000
+  // This flag was deprecated in CUDA 11
   if (getTensorCoreSupport(device)) {
     cublasSetMathMode(blasHandle, CUBLAS_TENSOR_OP_MATH);
   }
 #endif
+#if CUDA_VERSION >= 11000
+  cublasSetMathMode(blasHandle, CUBLAS_MATH_DISALLOW_REDUCED_PRECISION_REDUCTION);
+#endif
 
   FAISS_ASSERT(memory_.count(device) == 0);
 
-- 
2.26.2.windows.1

