From a6820516d2a0a704ef77f6df2d56ccb442238f88 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 26 May 2023 13:17:45 +1100
Subject: [PATCH 9/9] add missing headers

---
 faiss/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/faiss/CMakeLists.txt b/faiss/CMakeLists.txt
index cfb2b9b9..5f06b2b1 100644
--- a/faiss/CMakeLists.txt
+++ b/faiss/CMakeLists.txt
@@ -149,6 +149,7 @@ set(FAISS_HEADERS
   impl/AuxIndexStructures.h
   impl/CodePacker.h
   impl/IDSelector.h
+  impl/CodePacker.h
   impl/DistanceComputer.h
   impl/FaissAssert.h
   impl/FaissException.h
@@ -204,6 +205,7 @@ set(FAISS_HEADERS
   utils/simdlib_avx2.h
   utils/simdlib_emulated.h
   utils/simdlib_neon.h
+  utils/sorting.h
   utils/utils.h
   utils/distances_fused/avx512.h
   utils/distances_fused/distances_fused.h
