From 809eae41330972467fd30649f9b459c5f79d35c9 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Fri, 26 May 2023 13:17:45 +1100
Subject: [PATCH 11/11] add missing headers

---
 faiss/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/faiss/CMakeLists.txt b/faiss/CMakeLists.txt
index dccf5e95..0a264cdc 100644
--- a/faiss/CMakeLists.txt
+++ b/faiss/CMakeLists.txt
@@ -145,6 +145,7 @@ set(FAISS_HEADERS
   impl/AdditiveQuantizer.h
   impl/AuxIndexStructures.h
   impl/IDSelector.h
+  impl/CodePacker.h
   impl/DistanceComputer.h
   impl/FaissAssert.h
   impl/FaissException.h
@@ -196,6 +197,7 @@ set(FAISS_HEADERS
   utils/simdlib_avx2.h
   utils/simdlib_emulated.h
   utils/simdlib_neon.h
+  utils/sorting.h
   utils/utils.h
   utils/distances_fused/avx512.h
   utils/distances_fused/distances_fused.h
