From d49343b508f98e48dd34ece4d340f878b4a89d87 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 28 Feb 2021 00:00:57 +0100
Subject: [PATCH 13/14] switch correctly for AVX2 in swigfaiss-build

---
 faiss/python/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/faiss/python/CMakeLists.txt b/faiss/python/CMakeLists.txt
index 40ef40c1..2ed9c200 100644
--- a/faiss/python/CMakeLists.txt
+++ b/faiss/python/CMakeLists.txt
@@ -61,6 +61,10 @@ swig_add_library(swigfaiss
   SOURCES swigfaiss.swig
 )
 
+if(FAISS_OPT_LEVEL STREQUAL "avx2")
+  set_target_properties(swigfaiss PROPERTIES OUTPUT_NAME "swigfaiss_avx2")
+endif()
+
 if(NOT WIN32)
   # NOTE: Python does not recognize the dylib extension.
   set_target_properties(swigfaiss PROPERTIES SUFFIX .so)
-- 
2.29.2.windows.3

