From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: conda-forge <conda-forge@users.noreply.github.com>
Date: Thu, 2 Jul 2026 00:00:00 +1000
Subject: [PATCH] use CBLAS GEMM for Windows MKL

---
 faiss/utils/distances.cpp | 43 ++++++++++++++++++++++++-------------------
 1 file changed, 24 insertions(+), 19 deletions(-)

diff --git a/faiss/utils/distances.cpp b/faiss/utils/distances.cpp
index 0000000..0000000 100644
--- a/faiss/utils/distances.cpp
+++ b/faiss/utils/distances.cpp
@@ -18,6 +18,7 @@
 
 #include <omp.h>
 
+#include <cblas.h>
 #include <faiss/impl/AuxIndexStructures.h>
 #include <faiss/impl/FaissAssert.h>
 #include <faiss/impl/IDSelector.h>
@@ -31,25 +32,29 @@
 #define FINTEGER long
 #endif
 
-extern "C" {
-
-/* declare BLAS functions, see http://www.netlib.org/clapack/cblas/ */
-
-int sgemm_(
-        const char* transa,
-        const char* transb,
-        FINTEGER* m,
-        FINTEGER* n,
-        FINTEGER* k,
-        const float* alpha,
-        const float* a,
-        FINTEGER* lda,
-        const float* b,
-        FINTEGER* ldb,
-        float* beta,
-        float* c,
-        FINTEGER* ldc);
-}
+static int sgemm_(
+        const char* transa,
+        const char* transb,
+        FINTEGER* m,
+        FINTEGER* n,
+        FINTEGER* k,
+        const float* alpha,
+        const float* a,
+        FINTEGER* lda,
+        const float* b,
+        FINTEGER* ldb,
+        float* beta,
+        float* c,
+        FINTEGER* ldc) {
+    const CBLAS_TRANSPOSE ta = (transa[0] == 'T' || transa[0] == 't')
+            ? CblasTrans
+            : CblasNoTrans;
+    const CBLAS_TRANSPOSE tb = (transb[0] == 'T' || transb[0] == 't')
+            ? CblasTrans
+            : CblasNoTrans;
+    cblas_sgemm(CblasColMajor, ta, tb, *m, *n, *k, *alpha, a, *lda, b, *ldb, *beta, c, *ldc);
+    return 0;
+}
 
 namespace faiss {
 
