From 1c0e08310b0ab7afb558f987adfdf3bbcca3085c Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 18 Feb 2021 19:30:58 +0100
Subject: [PATCH 15/18] do not overload __builtin_ctz for win + cuda>=11

---
 faiss/impl/platform_macros.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/faiss/impl/platform_macros.h b/faiss/impl/platform_macros.h
index 1f4795c6..7efc9483 100644
--- a/faiss/impl/platform_macros.h
+++ b/faiss/impl/platform_macros.h
@@ -39,11 +39,14 @@ inline int __builtin_ctzll(uint64_t x) {
     return (int)ret;
 }
 
+// cudatoolkit provides __builtin_ctz for NVCC >= 11.0
+#if ! defined(__CUDACC__) || __CUDACC_VER_MAJOR__ < 11
 inline int __builtin_ctz(unsigned long x) {
     unsigned long ret;
     _BitScanForward(&ret, x);
     return (int)ret;
 }
+#endif
 
 inline int __builtin_clzll(uint64_t x) {
     return (int)__lzcnt64(x);
-- 
2.29.2.windows.3

